charts/jaeger/values.yaml

```diff
@@ -33,9 +33,19 @@ config:
     jaeger_storage:
```

Comment:
help me understand - this `config` structure is not actually used as a config, right? It only provides the shape for the properties that looks like OTEL config?